src/AppBundle/Controller/SheetController.php

```diff
@@ -51,6 +54,7 @@ public function newAction(Request $request)
         if ($form->isSubmitted() && $form->isValid()) {
             $em = $this->getDoctrine()->getManager();
             $sheet->setUser($this->getUser());
+            $sheet->setCreationDate($dateNow);
```

Comment:
Tu peux te passer de la variable $dateNow et faire direct le new DateTime() ici
ou mieux dans le construct de l'entité Sheet, tu fais un `$this->setDateTime(new \Datetime);`